addons-cluster/zookeeper/templates/_helpers.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "zookeeper-cluster.extra-envs" }}
```

Comment:
use `cluster.spec.componentSpecs.env`  instead, to inject envs for Cluster.

Reply:
done